hasjob/views/helper.py

```diff
@@ -599,3 +599,9 @@ def usessl(url):
     if url.startswith('http:'):  # http://www.example.com
         url = 'https:' + url[5:]
     return url
+
+
+@app.context_processor
+def for_filter_fields():
```

Comment:
- Ok. Then you can remove the corresponding variables that are being passed from `index.py`.
- Better function name perhaps? Maybe `inject_filter_choices`.